editor/editor_node.cpp

```diff
@@ -2289,7 +2289,7 @@ void EditorNode::_edit_current(bool p_skip_foreign) {
 		InspectorDock::get_inspector_singleton()->set_use_folding(!disable_folding);
 	}
 
-	/* Take care of PLUGIN EDITOR */
+	// Take care of PLUGIN EDITOR.
```

Comment:
This comment is odd. What is PLUGIN EDITOR?

Reply:
You're right, this is odd. We could change it to `// Take care of the main editor plugin.`